docker/addons/influxdb/docker-compose.yml

```diff
@@ -8,6 +8,10 @@
 
 version: "3"
 
+networks:
```

Comment:
Actually - should be `external` - like in other `writers`: https://github.com/mainflux/mainflux/blob/master/docker/addons/mongodb-writer/docker-compose.yml#L12

Reply:
Changes applied.

Reply:
Actually, `docker_` prefix is missing. I think it is important for Kubernetes.